.ci/install_runtime.sh

```diff
@@ -105,10 +105,7 @@ esac
 
 if [ x"${TEST_INITRD}" == x"yes" ]; then
 	echo "Set to test initrd image"
-	sudo sed -i -e '/^image =/d' "${runtime_config_path}"
-else
-	echo "Set to test rootfs image"
-	sudo sed -i -e '/^initrd =/d' "${runtime_config_path}"
+	sudo sed -i -e 's|^image =.*|initrd = "/usr/share/kata-containers/kata-containers-initrd.img"|' "${runtime_config_path}"
```

Comment:
This literal path might well cause trouble right now I'm afraid - based on my experience, kata might not always be in /usr/share.